### ISSUE TITLE
Start sharing on launch

### ISSUE THREAD

**Issue by andysavage** [onionshare/onionshare#1574]
This is a great resource and so easy to use compared to other offerings. Thanks.

I think it could be improved by adding a setting to automatically start sharing folders when launched. That would take care of situations where the server crashed and rebooted while nobody was present.

**Comment by ericsupreme** [comment]
agreed and to add on,  an option to have the share folder "refresh" when files are received would be great for teams etc.
